hooks/new

```diff
@@ -56,11 +56,20 @@ aws)
 
 		prompt_for aws_bosh_sgs multi-line \
 			'What security groups should the BOSH Director VM be in?'
-
-		prompt_for aws_key_name line \
-			'What is the name of the IAM SSH key to use for deployment?'
 	fi
-	;;
+
+  cat >&2 <<-"NOTE"
+  NOTE:
+    Before deploying please be sure to import the keypair generated for you from
+    vault into AWS console.
+    First run the following command to get the public key:
+      safe get secret/$prefix/aws/ssh:public
+    Then go to EC2 > Key Pairs > Import Key Pair and paste the value into the
+    Type 'vcap@$name' in the 'Key pair name' input box
+    Paste the output of the safe get command in the 'Public key contents' input box
+    Click 'Import' button, now you can deploy.
+NOTE
```

Comment:
A few minor nits:

1. Can you put an extra blank line after line 64, and after line 66?
2. It seems like you have extra sentence on line 67.
3. Perhaps number lines 65, 67-70 to make it clear that they are steps?